### ISSUE TITLE
Sentry Reports Suppressed Errors

### ISSUE THREAD

**Issue by mike-grinspan** [getsentry/sentry-laravel#201]
When running the `php artisan config:clear` Laravel attempts to delete the old cache file. If the file doesn't exist `unlink` emits a warning. To avoid that they use `@` to suppress it. Unfortunately this still gets sent to Sentry.

Seems related to getsentry/sentry-php#661

Installation code:

```php
public function report(Exception $exception) {
	if(config('sentry.dsn') && $this->shouldReport($exception)) {
		if(app()->runningInConsole()) {
			$user = ['id' => 'console'];
		} elseif(auth()->check()) {
			$user = auth()->user();

			$user = [
				'email'    => $user->email,
				'id'       => $user->id,
				'name'     => $user->name,
				'username' => $user->username,
			];
		} else {
			$user = ['id' => 'unauthenticated'];
		}

		/** @var \Sentry\State\Hub $sentry */
		$sentry = app('sentry');

		$sentry->configureScope(function (Scope $scope) use (&$user) {
			$scope->setUser($user);
		});

		if(file_exists(base_path('release'))) {
			$sentry->getClient()
				->getOptions()
				->setRelease(trim(file_get_contents(base_path('release'))));
		}

		$sentry->captureException($exception);
	}

	parent::report($exception);
}
```

**Comment by stayallive** [comment]
Can you let me know on what OS and PHP version you are running? I can't ssem reproduce the behaviour. 

I also installed `1.0.0` in a clean Laravel and made sure it can send errors and ran `php artisan config:clear` a few times without result 😞 

As a side note, a `warning` error does not go through the report function of the exception handler. So the installation code you pasted should do nothing, and the tags/user context should not have been set. But in any case ofcourse a `@unlink` should not ben sent to Sentry, so still eager to find the culprit!

**Comment by mike-grinspan** [comment]
We're running Ubuntu 18.04.2 and PHP 7.2.15

I didn't think the installation code would matter but supplied it as you asked the other guy for it in getsentry/sentry-php#661

**Comment by stayallive** [comment]
@mike-grinspan thanks for the extra info! And yes because that was a plain-php installation not Laravel, but better to have too much info than too little!

Hope with all the info (also from #199) we can finally find this issue!

**Comment by stayallive** [comment]
@mike-grinspan This has been fixed in `2.0.1` of the base SDK (`sentry/sentry`) that was just released, running `composer update` should pull that in and fix this issue. 

A new option has been added (`capture_silenced_error`) to configure whether suppressed errors should be captured by Sentry and by default this is `off`.
